tools/fast/fastesting/basic.go

```diff
@@ -138,3 +142,22 @@ func (env *TestEnvironment) RequireNewNodeWithFunds(funds int) *fast.Filecoin {
 
 	return p
 }
+
+func (env *TestEnvironment) Teardown(ctx context.Context) error {
```

Comment:
Not sure if the linter caught this, but please comment these methods

Reply:
Ya, we had this issue the last time. I'm really curious why the linter isn't catching this stuff.

Reply:
@travisperson could you please look into linter behaviour for us? We may have lost something during gomod migration.